docs/book/src/topics/custom-images.md

```diff
@@ -76,7 +76,7 @@ spec:
   template:
     spec:
       image:
-        sharedGallery:
+        computeGallery:
```

Comment:
>I think this should stay sharedGallery in the example.

This PR deprecates this field and `computeGallery` provides the same functionality if using SIG images without plan information. I think we shouldn't be promoting deprecated fields anymore.

At the same time, I don't think community galleries should be _recommended_ at the moment, as they are still in preview.

Could you also please elaborate what has #2293 to do with this PR? I can't see the connection?

Reply:
sorry the linked didn't copy paste properly, it should have been a link to a few lines above this section on line 66: https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2277/files#diff-1bff25aa275cb44ca787e388c1774fcc0663c2be151dbbe9f3e44c6325a1fd30L66

> This PR deprecates this field and computeGallery provides the same functionality if using SIG images without plan information. I think we shouldn't be promoting deprecated fields anymore.

ok this makes sense. I was getting `Compute Image gallery` and` Community Galleries` conflated.  Maybe this section should be updated to `### Using Compute Image Gallery (Recommended)` instead of `### Using Shared Image Gallery (Recommended)` and add a note that it was renamed from SIG -> Compute Image Gallery

Reply:
>Maybe this section should be updated to ### Using Compute Image Gallery (Recommended) instead of ### Using Shared Image Gallery (Recommended) and add a note that it was renamed from SIG -> Compute Image Gallery

Yes, that definitely make sense. Perhaps something like:
```md
### Using Compute Gallery (Recommended)

To use an image from the [Compute Gallery][compute-gallery], previously known as Shared Image Gallery (SIG), fill in the `resourceGroup`, `name`, `subscriptionID`, `gallery`, and `version` fields:
```

I'm not sure if a warning or a dedicated note about the rename is needed here.

Reply:
lgtm, that way someone searching for the previous name will still find it

Reply:
Fixed, so marking as resolved.